libs/ledger-live-common/src/featureFlags/defaultFeatures.ts

```diff
@@ -122,6 +122,63 @@ export const defaultFeatures = {
   editEthTx: {
     enabled: false,
   },
+  npsRatingsPrompt: {
```

Comment:
instead of creating a new feature flag maybe there should be only 1 flag that configures whether we use the NPS flow or the old flow?